### PR TITLE
[issues] styling and ally issues for release

### PR DESCRIPTION
**Pull request by philli-m**

### CODE REVIEW

**Comment by philli-m** [comment]
@goapunk I did not manage to disable the local csp to test capcha, if you have any suggestions happy to accept

**Comment by goapunk** [comment]
@philli-m the rating styling looks off

File: meinberlin/assets/scss/components/_list_item.scss [hunk 1]
```diff
@@ -130,20 +137,3 @@
...
-    .list-item__vote {
-        display: flex;
-        justify-content: space-between;
-        align-items: center;
-    }
-
-    .list-item__header {
-        flex-direction: row;
-    }
-}
```
Comment (philli-m) [inline meinberlin/assets/scss/components/_list_item.scss]: @goapunk this commit for some reason reset the other changes from the first commit, no clue why?!

**Review by goapunk — CHANGES_REQUESTED** [review]
Looks good, only a small typo. I couldn't reproduce https://github.com/liqd/a4-meinberlin/issues/5171 either so will have to see.

File: meinberlin/assets/scss/components/_list_item.scss [hunk 2]
```diff
@@ -137,3 +130,21 @@
...
+@media screen and (min-width: $breakpoint-xs) {
+    .list-item__vote {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    .list-item__header {
+        flex-direction: row;
+        flex-direction: space-between;
```
[the remark this answers is not in the document]
Comment (philli-m) [reply meinberlin/assets/scss/components/_list_item.scss:148]: yes...

**Review by goapunk — APPROVED** [review]
cool, thanks for all fixes!

**Pull request merged**